lib/assessment.rb

```diff
@@ -13,28 +13,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(&:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? { |x| x.odd? }
+    true
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.all? { |x| x.even? }
+    true
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    words.capitalized { |words| }
```

Comment:
Cramsters, things get squirrelly without Ruby Docs on hand.  I need
something that looks at every instance of the array and I would probably
put .capitalize after the block?
On Oct 23, 2015 7:23 PM, "Matt Brendzel" <notifications@github.com> wrote:

> In lib/assessment.rb
> <https://github.com/ga-wdi-boston/ruby-enumerable-assessment/pull/10#discussion_r42924496>
> :
>
> >    end
> >
> >    # Question 5
> >    # Write ruby code that returns an array with all of the
> >    # entries from the words enumerable capitalized.
> >    def transform(words)
> > -    # Your code goes here
> > +    words.capitalized { |words| }
>
> Not quite. You want to perform a transformation across all the elements in
> words - what do you think the right tool would be for that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/ruby-enumerable-assessment/pull/10/files#r42924496>
> .
>